opcua/server/internal_subscription.py

```diff
@@ -140,10 +142,12 @@ def _delete_monitored_items(self, mid):
 
     def datachange_callback(self, handle, value, error=None):
         if error:
-            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'", self, handle, error)
+            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'",
```

Comment:
OK let says we use 120, this seems rather common.

On Sun, May 8, 2016, 09:29 Denis Štogl <notifications@github.com> wrote:

> In opcua/server/internal_subscription.py
> <https://github.com/FreeOpcUa/python-opcua/pull/179#discussion_r62428171>:
>
> > @@ -140,10 +142,12 @@ def _delete_monitored_items(self, mid):
> >
> >      def datachange_callback(self, handle, value, error=None):
> >          if error:
> > -            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'", self, handle, error)
> > +            self.logger.info("subscription %s: datachange callback called with handle '%s' and erorr '%s'",
>
> But 80 characters length according to PEP8 I find not soo cool. I would
> than rather agree with 120 or something like that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/179/files/0c708a46a89e425c33fa4e03a6cf5b4225f95cfe..4cb7ce42945cb12906be0e93c0165a72dbd1229b#r62428171>
>